src/ts/components/forms/form-group.examples.md

```diff
@@ -56,6 +56,9 @@
 #### Less variables
 
 ```less
+@form-group-margin: @margin-large;
+@form-group-label-margin: @margin-base;
```

Comment:
Please copy and paste all variables back in here, in the same order.

Reply:
I didn't remove them, there are just not in the picture. Let me update the screenshot.

Reply:
> in the same order.

Reply:
Oh, this isn't all the variables. You need to update the ALL variables documentation.

Reply:
Wait, no you don't. My bad. I forgot how this all works.